.github/workflows/main.yml

```diff
@@ -5,23 +5,21 @@ on:
   push:
     branches:
       - master
-    tags:
-      - v*
   pull_request:
     branches:
       - master
   workflow_dispatch:
 
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.7
+      - name: Set up python 3.9
         uses: actions/setup-python@v3
```

Comment:
```suggestion
        uses: actions/setup-python@v4
```

Reply:
Why do not push latest version to [actions/setup-python](https://github.com/actions/setup-python/releases/tag/v4.0.0) ?